### PR TITLE
Feat/breadcrumbs

### PR DESCRIPTION
**Pull request by Lamperoyge**
## :pushpin: References

- **Wonder issue:**
- **Related pull-requests:**

## :movie_camera: Screenshot or Video

CHANGES:

1. Header is now 36px instead of 42
2. Breadcrumbs
3. Removed space below the header on mobile
4. Search is now on the right side 

https://www.loom.com/share/8ff6abb031d74d4eb1620b875a875d66

## :cake: Checklist:

- [x] I self-reviewed my changes
- [x] My changes follow the style guide: https://creative-earth-33e.notion.site/TT-Wonder-Style-guide-4702a45133374148953bfcaf584120b7
- [x] I tested my changes in Chrome

### CODE REVIEW

File: wondrous-app/components/BreadCrumbs/Components/CollabSelector.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  const { pageData } = useGlobalContext();
+  const activePodOrg = useMemo(() => pageData?.pod?.org, [pageData?.pod]);
+
+  const activeOrg = pageData?.orgData;
+
+  const collab = activeOrg || activePodOrg || {};
+
+  return (
+    <Grid display="flex" gap="14px" justifyContent="center" alignItems="center">
+      {collab?.parentOrgs?.map((org, idx) => (
```
Comment (andywong418) [inline wondrous-app/components/BreadCrumbs/Components/CollabSelector.tsx:20]: Nit: Technically don't need the ternary operator on collab

File: wondrous-app/components/BreadCrumbs/Components/PodSelector.tsx [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+  useEffect(() => {
+    if (open && !data) {
+      getUserPods({
+        variables: {
+          orgId: activePod?.orgId,
+        },
+      });
+    }
+  }, [open]);
```
Comment (andywong418) [inline wondrous-app/components/BreadCrumbs/Components/PodSelector.tsx:54]: Isn't there a dependency on `data` here as well?

File: wondrous-app/components/BreadCrumbs/Components/PodSelector.tsx [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+              value={selectedValue}
+              renderInput={(params) => (
+                <PodSearchInput
+                  {...params}
+                  ref={params.InputProps.ref}
+                  disableUnderline
+                  fullWidth
+                  autoFocus
+                  InputProps={{
+                    ...params.InputProps,
```
Comment (andywong418) [inline wondrous-app/components/BreadCrumbs/Components/PodSelector.tsx:85]: Same questions on ternary

File: wondrous-app/components/BreadCrumbs/Components/PodSelector.tsx [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+              )}
+              PaperComponent={PodSearchPaper}
+              ListboxComponent={PodSearchList}
+              PopperComponent={(params) => <PodSearchAutocompletePopper {...params} />}
+              open={open}
+              options={data?.getAvailableUserPods ?? []}
+              disablePortal
+              onChange={(event, value, reason) => {
+                if (reason === 'selectOption') {
+                  const searchParams = new URLSearchParams(window.location.search);
```
Comment (andywong418) [inline wondrous-app/components/BreadCrumbs/Components/PodSelector.tsx:111]: Any reason we're using `window.location` instead of `router`?

**Review by andywong418 — COMMENTED** [review]
Great work! Just some questions from me

File: wondrous-app/components/BreadCrumbs/Components/CollabSelector.tsx (hunk 1 above)
Comment (Lamperoyge) [reply wondrous-app/components/BreadCrumbs/Components/CollabSelector.tsx]: true, fixed

File: wondrous-app/components/BreadCrumbs/Components/PodSelector.tsx (hunk 2 above)
Comment (Lamperoyge) [reply wondrous-app/components/BreadCrumbs/Components/PodSelector.tsx]: yep, fixed

File: wondrous-app/components/BreadCrumbs/Components/PodSelector.tsx [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+        <PodSearchPopper open={open} anchorEl={anchorEl} placement="bottom-start" disablePortal>
+          {loading ? (
+            <Spinner />
+          ) : (
+            <PodSearchAutocomplete
+              value={selectedValue}
+              renderInput={(params) => (
+                <PodSearchInput
+                  {...params}
+                  ref={params.InputProps.ref}
```
[the remark this answers is not in the document]
Comment (Lamperoyge) [reply wondrous-app/components/BreadCrumbs/Components/PodSelector.tsx:79]: we don't really, params always exists as 

  id: string;
  disabled: boolean;
  fullWidth: boolean;
  size: 'small' | undefined;
  InputLabelProps: ReturnType<ReturnType<typeof useAutocomplete>['getInputLabelProps']>;
  InputProps: {
    ref: React.Ref<any>;
    className: string;
    startAdornment: React.ReactNode;
    endAdornment: React.ReactNode;
  };
  inputProps: ReturnType<ReturnType<typeof useAutocomplete>['getInputProps']>;

File: wondrous-app/components/BreadCrumbs/Components/PodSelector.tsx [hunk 6]
```diff
@@ -0,0 +1,133 @@
...
+                    ),
+                  }}
+                />
+              )}
+              disableClearable
+              isOptionEqualToValue={(option, value) => option.id === value?.id}
+              getOptionLabel={(option) => option.name}
+              renderOption={(props, option) => (
+                <PodSearchListItem {...props} isActive={option.id === selectedValue?.id}>
+                  <PodSearchDefaultImage color={option?.color ?? '#474747'} />
```
[the remark this answers is not in the document]
Comment (Lamperoyge) [reply wondrous-app/components/BreadCrumbs/Components/PodSelector.tsx]: nice, fixed

File: wondrous-app/components/BreadCrumbs/Components/PodSelector.tsx (hunk 4 above)
Comment (Lamperoyge) [reply wondrous-app/components/BreadCrumbs/Components/PodSelector.tsx]: changing entityType on boards page doesn't trigger a router state update, only changes the url in case the user wants to copy the url with the entityType inside, the logic behind this was to basically  keep the entityType on pod change, so if the user is in pod / podId / bounties , on podId change keep the user on the bounties. But on a second thought looks hackish, changed it now while keeping the same functionality

Comment (andywong418) [reply wondrous-app/components/BreadCrumbs/Components/PodSelector.tsx]: Gotcha makes sense!

File: wondrous-app/pages/collaboration/[username]/boards.lazy.tsx [hunk 7]
```diff
@@ -385,6 +386,14 @@ function BoardsPage() {
     filters,
   });
 
+  useEffect(() => {
```
Comment (terryli0095) [inline wondrous-app/pages/collaboration/[username]/boards.lazy.tsx:389]: is this type of pattern common?

**Review by terryli0095 — APPROVED** [review]

File: wondrous-app/pages/collaboration/[username]/boards.lazy.tsx (hunk 7 above)
Comment (Lamperoyge) [reply wondrous-app/pages/collaboration/[username]/boards.lazy.tsx:389]: It’s a parent state update from the child, since our header requires data from the inside of the page we can either update it from here or make a new fetch request inside the header, but even then we still rely on the current entityType to render breadcrumbs. Another way to do this is to lift state and main request to parent component, this can work too but over time can be hard to manage imo, we can also move Header inside the page but we would get a unmount->mount on every page change and it would cause flickering on static elements inside the header

tldr:  I think it’s okay and the pattern is pretty common tho not ideal imo
